### PR TITLE
Add documentation on how to do releases

### PR DESCRIPTION
**Pull request by marvinhagemeister**
May be useful for other contributors with publishing rights.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+5. Run `npm run build && npm publish`
+   1. Make sure you have 2FA enabled in npm, otherwise the above command will fail.
+   2. If you're doing a pre-release add `--tag next` to the `npm publish` command to publish it under a different tag (default is `latest`)
+6. Publish the release notes and create the correct git tag.
+7. Tweet it out
+
+## Legacy Releases (8.x)
+
+> **ATTENTION:** Make sure that you've cleared the project correctly
+> when switching from a 10.x branch.
```
Comment (developit) [inline CONTRIBUTING.md]: Suggestion: `rm -rf dist node_modules && npm i`

**Comment by developit** [comment]
awesome awesome

File: CONTRIBUTING.md (hunk 1 above)
Comment (marvinhagemeister) [reply CONTRIBUTING.md]: Added :tada:

**Comment by andrewiggins** [comment]
This is great. Thanks!!

**Pull request merged**